lib/stripe/core_resources/customer.ex

```diff
@@ -93,7 +89,7 @@ defmodule Stripe.Customer do
                  optional(:metadata) => Stripe.Types.metadata(),
                  optional(:shipping) => Stripe.Types.shipping(),
                  optional(:source) => Stripe.id() | Stripe.Source.t(),
-                 optional(:tax_info) => Stripe.Types.tax_info(),
+                 optional(:tax_id_data) => Stripe.TaxID.tax_id_data(),
```

Comment:
Looks like you can also pass an `address` to the [create](https://stripe.com/docs/api/customers/create) method.  What would you like to do about this (and others)?  Follow up PR?